packages/prisma/schema.prisma

```diff
@@ -615,6 +615,16 @@ model WorkflowsOnEventTypes {
   eventTypeId Int
 }
 
+model Deployment {
+  id               Int                    @id @default(1)
```

Comment:
This change is missing an extra migration. Will open up another PR to migrate first.

Reply:
https://github.com/calcom/cal.com/pull/6706